lib/sparrow/h2_worker.ex

```diff
@@ -78,8 +78,16 @@ defmodule Sparrow.H2Worker do
   end
 
   def handle_info(:ping, state) do
-    H2Adapter.ping(state.connection_ref)
-    _ = schedule_message_after(:ping, state.config.ping_interval)
+    _ =
```

Comment:
Why bind to wildcard here?

Reply:
wildcard because I have https://github.com/esl/sparrow/blob/4e61414546cd0a2bcd0eaa3df6a7a58941ee3277/mix.exs#L50 set in dialyzer, but I I'll fix all of the wildcards in separate PR